scipy/stats/tests/test_rank.py

```diff
@@ -192,6 +192,53 @@ def test_size_0_axis(self, axis, method, dtype):
         assert_equal(r.shape, shape)
         assert_equal(r.dtype, dtype)
 
+    def test_nan_policy(self):
+        # 1 1d-array test
+        # 1-1 propagate test
+        assert_array_equal([2., 3., 4., 1., 5., 6.],
+                           rankdata([0, 2, 3, -2, np.nan, np.nan],
+                                    nan_policy="propagate"))
```

Comment:
```suggestion
        data = [0, 2, 3, -2, np.nan, np.nan]
        res = rankdata(data, nan_policy="propagate")
        assert_array_equal(res, [2., 3., 4., 1., 5., 6.])
```
`data` can be re-used below.
The order of inputs to `assert_array_equal` doesn't really matter in practice, but this reverses them to adhere to the convention:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/6570539/167302809-497ffdd9-c583-49c1-a376-3c87f62eee0d.png">

This is also nice because there is no more line continuation / looks cleaner IMO.